clients/web/test/testUtils.js

```diff
@@ -670,6 +658,32 @@ girderTest.importStylesheet = function (css) {
 };
 
 /**
+ * Import the JS and CSS files for a plugin.
+ */
+girderTest.importPlugin = function (pluginName) {
```

Comment:
It would be nice to allow this to import a list of plugins, either by passing a list, or via `arguments`.